src/js/constants.js

```diff
@@ -1,4 +1,39 @@
 export const THEME = {
+  buttonDisabled: {
+    backgroundColor: "var(--white)",
+    borderColor: "var(--gray-3)",
+    fillColor: "var(--gray-3)",
+  },
+  primaryButtonDefault: {
+    backgroundColor: "var(--orange-4)",
+    borderColor: "var(--black)",
+    fillColor: "var(--black)",
+  },
+  primaryButtonHover: {
+    backgroundColor: "var(--gray-2)",
+    borderColor: "var(--white)",
+    fillColor: "var(--white)",
+  },
+  primaryButtonSelected: {
+    backgroundColor: "var(--gray-2)",
+    borderColor: "var(--white)",
+    fillColor: "var(--white)",
+  },
+  secondaryButtonDefault: {
+    backgroundColor: "var(--gray-2)",
+    borderColor: "var(--white)",
+    fillColor: "var(--white)",
+  },
+  secondaryButtonHover: {
+    backgroundColor: "var(--orange-4)",
+    borderColor: "var(--black)",
+    fillColor: "var(--black)",
+  },
+  secondaryButtonSelected: {
+    backgroundColor: "var(--orange-4)",
+    borderColor: "var(--black)",
+    fillColor: "var(--black)",
+  },
```

Comment:
Can we keep it simple like it was. We start by the states `disabled` `primary` `hover` etc... and then give it color that is most used (I think at most 1 or 2 colors for each states) and we only introduce component variations/naming when we need to. Like `icon`, `button`, etc...

Reply:
I'm not sure I understand what you're saying. I'll push some code where I factor out the functions, but I think it was simpler before because we only had one type of button (and we need two based on Andrea's mock-ups).

Reply:
something like `{default: {standardButton: {...}, iconButton: {...}}, hover: {}, error: {border: "red", color: "red", background: "white"}, disabled {...}}` since a lot of the components share colors in the other states. To make it reusable maybe later.

Reply:
Ah, I see. I like that. I'll make the update to `constants.js`